api/v1beta2/ibmpowervscluster_webhook.go

```diff
@@ -209,7 +216,7 @@ func (r *IBMPowerVSCluster) validateIBMPowerVSClusterCreateInfraPrereq() (allErr
 	if r.Spec.ResourceGroup == nil {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("spec.resourceGroup"), r.Spec.ResourceGroup, "value of resource group is empty"))
 	}
-	if err := r.validateIBMPowerVSClusterVPCSubnetNames(); err != nil {
+	if err := r.validateIBMPowerVSClusterVPCSubnets(); err != nil {
```

Comment:
Since we are only validating the subnet name duplication, Can we revert back to old change? Can we directly call validateIBMPowerVSClusterVPCSubnetNames() from here?

Reply:
Done